src/storage/aggregator.cc

```diff
@@ -786,82 +821,128 @@ bool MinMaxBaseAggregator::DecodeAggrVal(const int8_t* row_ptr, AggrBuffer* buff
     return true;
 }
 
+bool MinMaxBaseAggregator::UpdateAggrVal(const codec::RowView& row_view, const int8_t* row_ptr, AggrBuffer* aggr_buffer,
+                   const std::string& key, bool reverse) {
+    return true;
+}
+
+bool MinMaxBaseAggregator::RebuildExtremumAggrBuffer(const codec::RowView& row_view,
+                                                     AggrBuffer* aggr_buffer,
+                                                     const std::string& key) {
+    if (base_table_ == nullptr) {
+        PDLOG(ERROR, "base table is nullptr, cannot update MinAggr table");
+        return false;
+    }
+    auto it = base_table_->NewTraverseIterator(this->GetIndexPos());
```

Comment:
if you prefer to use `TraverseIterator`, you have to check the `key` and `GetPK` are the same. So here better to use `WindowIterator` to simplify the code and performance concern.